This is awesome.  In RHEL, I needed to add `-ldl` to the `EXTRA_LIBS` in order to resolve:
```
/usr/bin/ld: atbridge/pcap_delay.o: undefined reference to symbol 'dlclose@@GLIBC_2.2.5'
```
Does adding that help or hinder your build?

--- a/src/Makefile
+++ b/src/Makefile
@@ -62,7 +62,7 @@ gsportmac: $(OBJECTS) compile_time.o
 
 # Linux for X builds:
 gsportx: $(OBJECTS) compile_time.o
-	$(LD) $(CCOPTS) $(LDOPTS) $(OBJECTS) compile_time.o $(LDFLAGS) -o $(NAME)$(SUFFIX) $(XLIBS) $(EXTRA_LIBS) -lX11
+	$(LD) $(CCOPTS) $(LDOPTS) $(OBJECTS) compile_time.o $(LDFLAGS) -o $(NAME)$(SUFFIX) $(XLIBS) $(EXTRA_LIBS) -lX11 -ldl
 	echo $(OBJECTS)
 	mv gsportx ..
 	cp -f ../config.template ../config.txt 

